ios-base/DroidKaigi 2020/FloorMap/FloorMapViewController.swift

```diff
@@ -29,19 +29,39 @@ final class FloorMapViewController: ContentViewController {
 // MARK: - Private functions
 
 private extension FloorMapViewController {
+    
+    /// This method caches the floor map image in LRU memory when loading from server.
     func loadMap() {
-        let width = imageView.image?.size.width
-        let height = imageView.image?.size.height
-
+        // Configure cache
+        ImageCache.shared.costLimit = 1024 * 1024 * 100 // 100 MB
+        ImageCache.shared.countLimit = 100
+        ImageCache.shared.ttl = 120 // Invalidate image after 120 sec
```

Comment:
During [the event](https://droidkaigi.jp/2020/en/) of the day,
I think that I look at the timeline and check the floor map many times in a short time.
So, I adjusted the cache time to 120 seconds.
If you have any other ideas, please let me know!😊

Reply:
I think it is ok. 😄
But I think `costLimit` and `countLimit` are too large value for this. 😭